tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1865,9 +1865,9 @@ Here is an example:
     var my_file_ref
 
     func _ready():
-        var f = File.new()
+        var f = FileAccess.new()
```

Comment:
FileAccess can't be directly instantiated, using something like `FileAccess.open("user://example_file.json", FileAccess.READ)` returns a FileAccess reference.

Reply:
Done!